scripts/build-soracom/fetch_plugins.sh

```diff
@@ -69,11 +104,11 @@ yarn_build_repo () {
   fi
 }
 
-clone_private_repo soracom-harvest-backend main
-clone_private_repo soracom-map-panel lagoon3-master
-clone_private_repo soracom-image-panel lagoon3-master
-clone_private_repo soracom-plot-panel main
-clone_private_repo soracom-dynamic-image-panel lagoon3-master
+clone_private_repo soracom-harvest-backend HASH_PLACEHOLDER
```

Comment:
The plan is to specify commit hashes here like in https://github.com/soracom/grafana/blob/soracom-shogo-deploy-experiment/scripts/build-soracom/fetch_plugins.sh#L107

I'm setting these to placeholder because add_deployment_stuff will be around for a long time, and I don't want us to accidentally re-merge this in in the future and run into an issue where the commit hashes go back to an old version of the plug-ins